### PR TITLE
Correct the properties used at ranger authorizer

### PR DESCRIPTION
**Pull request by zemin-piao**
### Description

Following changes are done at `druid-ranger-security.md`

Correct the default value of `use_ugi`.

Correct the properties used for ranger authorizer.

#### Release note

For tips about how to write a good release note, see [Release notes](https://github.com/apache/druid/blob/master/CONTRIBUTING.md#release-notes).

<hr>

##### Key changed/added classes in this PR
 * docs/development/extensions-core/druid-ranger-security.md

<hr>

This PR has:

- [x] been self-reviewed.
- [x] added documentation for new or modified features or behaviors.
- [x] been tested in a test Druid cluster.

### CODE REVIEW

**Comment by FrankChen021** [comment]
This change reflects current code. 
https://github.com/apache/druid/blob/72c16097ac6c04915f0137a1a37fed19f597799b/server/src/main/java/org/apache/druid/server/initialization/AuthorizerMapperModule.java#L115


And the original doc was introduced in #9579 which has not been changed since then, @bolkedebruin Could you make a confirmation on this change?

**Review by techdocsmith — COMMENTED** [review]
Changes from docs perspective LGTM. Recommend additional +1 for technical side @FrankChen021  or @bolkedebruin

Thanks for contributing the updates @zemin-piao

**Review by ektravel — APPROVED** [review]
LGTM

File: helm/druid/Chart.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 apiVersion: v2
-appVersion: 0.23.0
```
Comment (abhishekagarwal87) [inline helm/druid/Chart.yaml:17]: can you revert these yaml files? I can merge the PR afterwards.

**Pull request closed without merging**